User-Service/Controllers/UserController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using User_Service.Dtos;
+using User_Service.Interfaces;
+using User_Service.Models;
+using User_Service.Services;
+
+namespace User_Service.Controllers
+{
+    /// <summary>
+    /// Property Controller
+    /// </summary>
+    [Route("api/[controller]")]
```

Comment:
I think we probably want this to be.
```suggestion
  [Route("organizations")]
```
The api portion should already be added by the load balancer. Right now, we would require calling 
`api/api/[UserController]`